warehouse/models/mart/gtfs/_mart_gtfs.yml

```diff
@@ -1453,3 +1496,130 @@ models:
+          - not_null
+      - name: service_date
+        description: Date on which this stop was active.
+      - name: feed_key
+        description: |
+          Foreign key to the `dim_schedule_feeds` table.
+      - name: stop_id
+        description: '{{ doc("gtfs_stops__stop_id") }}'
+      - name: stop_event_count
+        description: The number of stop events associated with this stop on the given date.
+      - name: route_type_0_count
+        description: |
+          The count of stop events associated with route_type 0 - Tram, Streetcar, Light rail
+      - name: route_type_1_count
+        description: |
+          The count of stop events associated with route_type 1 - Subway, Metro
+      - name: route_type_2_count
+        description: |
+          The count of stop events associated with route_type 2 - Rail
+      - name: route_type_3_count
+        description: |
+          The count of stop events associated with route_type 3 - Bus
+      - name: route_type_4_count
+        description: |
+          The count of stop events associated with route_type 4 - Ferry
+      - name: route_type_5_count
+        description: |
+          The count of stop events associated with route_type 5 - Cable Tram
+      - name: route_type_6_count
+        description: |
+          The count of stop events associated with route_type 6 - Aerial lift, suspended cable car (e.g., gondola lift, aerial tramway)
+      - name: route_type_7_count
+        description: |
+          The count of stop events associated with route_type 7 - Cable Tram
+      - name: route_type_11_count
+        description: |
+          The count of stop events associated with route_type 11 - Trolleybus
+      - name: route_type_12_count
+        description: |
+          The count of stop events associated with route_type 12 - Monorail
+      - name: contains_warning_duplicate_stop_times_primary_key
+        description: |
+          Rows with `true` in this column indicate that the columns in this table that are aggregated from
+          stop times data (`n_stops`, `n_stop_times`,  `trip_first_departure_sec`, `trip_last_arrival_sec`,
+          and `service_hours`) contain at least one row that had a duplicate primary key in the source stop times data.
+
+          I.e., at least one row being aggregated had a `trip_id` / `stop_sequence` pair that was not unique
+          in the input data. This indicates that data quality issues were present in the stop times data
+          that is being summarized here, and counts may be inflated due to multiple rows with identical identifiers.
+      - name: contains_warning_duplicate_trip_primary_key
+        description: |
+          Rows with `true` in this column have a duplicate primary key in dim_trips;
+          i.e., `trip_id` is duplicated within an individual feed instance.
+          Treat these rows with caution.
+      - name: contains_warning_missing_foreign_key_stop_id
```

Comment:
If you're joining on `stop_id`, is it possible for this to be true? Are there rows in this table with a null stop_key? If so I think we need to rethink the merge order -- if this is supposed to be a stop-level fact, it shouldn't have missing keys at that level.

Reply:
okay @lauriemerrell so there were null `stop_key`s on this table, in my latest commit I just revised the join and the join order with `dim_stops` but would love your thoughts on that as a solution when you have a moment, I also revised the descriptions of the `warning_*` columns, and their use in tests

Reply:
Yes, I think this is correct 👍